packages/odyssey-storybook/src/guidelines/Roadmap/roadmapData.tsx

```diff
@@ -0,0 +1,220 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Status } from "@okta/odyssey-react-mui";
+import { DataColumn } from "@okta/odyssey-react-mui/labs";
+
+export type OdysseyComponent = {
+  name: string;
+  status: "Fully released" | "In Labs" | "In progress" | "Not started";
+  startDate: string;
+  labsRelease: string;
+  fullRelease: string;
+};
+
+export const columns: DataColumn[] = [
+  {
+    accessorKey: "name",
+    header: "Name",
+    enableHiding: false,
+  },
+  {
+    accessorKey: "status",
+    header: "Status",
+    Cell: ({ cell }) => {
+      const value = cell.getValue<string>();
+      const severity =
+        value === "Fully released"
+          ? "success"
+          : value === "In Labs"
+          ? "warning"
+          : value === "In progress"
+          ? "default"
+          : value === "Not started"
+          ? "error"
+          : "default";
+      return <Status label={value} severity={severity} variant="pill" />;
+    },
+  },
+  {
+    accessorKey: "startDate",
+    header: "Start date",
+    Cell: ({ cell }) => {
+      return cell.getValue() ? cell.getValue() : "—";
+    },
```

Comment:
A lot of repeated code below... perhaps extract this function out to a variable for reuse?